crates/sovereign-sdk/rollup-interface/src/state_machine/soft_confirmation.rs

```diff
@@ -181,6 +181,11 @@ impl SignedSoftConfirmation {
         self.signature.clone()
     }
 
+    /// Signature of the sequencer as ref
+    pub fn signature_as_ref(&self) -> &[u8] {
```

Comment:
```suggestion
    pub fn signature(&self) -> &[u8] {
```

Reply:
i think there is also a method named `signature` right above